ref/general/development-mode.adoc

```diff
@@ -62,31 +64,28 @@ Maven: `mvn liberty:dev -DhotTests`
 
 Gradle: `gradle libertyDev --hotTests`
 
-You can also add parameters to specify whether to skip unit or integration tests, or skip tests altogether.
+You can also add parameters to specify whether to skip tests.
+For Maven, you can add parameters to skip unit tests, skip integration tests, or skip both tests. For Gradle, you can add a parameter to skip all tests.
```

Comment:
"For Maven, you can add parameters to skip unit tests, skip integration tests, or skip both tests. For Gradle, you can add a parameter to skip all tests"
Add 'the'
"For Maven, you can add parameters to skip unit tests, skip integration tests, or skip both the tests. For Gradle, you can add a parameter to skip all the tests"